docs/en/ingest-management/getting-started-traces.asciidoc

```diff
@@ -61,11 +60,16 @@ In a later step, you'll instrument your code with APM agents and send the data
 to an APM Server instance that {agent} spins up.
 ====
 
-include::elastic-agent/elastic-agent.asciidoc[tag=agent-install-intro]
+{agent} is a single, unified agent that you can deploy to hosts or containers to
+collect data and send it to the {stack}. Behind the scenes, {agent} runs the
+{beats} shippers or Elastic Endpoint required for your configuration.
 
 If you plan on enabling Real User Monitoring (RUM), you must run {agent} centrally.
 If RUM is disabled, you should run {agent} on edge machines.
 
+//QUESTION FOR BRANDON: Should we use an include here like we do in the logs
```

Comment:
@bmorelli25 Note my question here. I think originally we decided to keep things simple by duplicating content, but this section changes too much. It would be better to pull the steps from the official installation docs (like I do in the other QS). I've cleaned up other duplicated info so that we only need to maintain this content in one place now. Let me know what you think.

Reply:
Ah, I missed this. Yes! Agreed. It made sense to duplicate in the beginning, but no longer. Let's do it.